app/controllers/ResponseController.scala

```diff
@@ -76,11 +229,19 @@ class ResponseController @Inject()(
         .fold(
           hasErrors => renderer.render("response.njk", json(hasErrors)).map(BadRequest(_)),
           (value: ResponseModel) => {
-            println(s"************* ArrivalId ${value.arrivalId} ************")
-            println(s"************* Version ${value.version} *************")
-            println(s"************* messageType ${value.messageType} *************")
-            destinationConnector.goodsReleased(goodsReleasedXml, value.arrivalId, value.version)
-            Future.successful(Redirect(routes.ResponseController.onPageLoad()))
+            value.messageType match {
+              case x if (x == "1" || x == "2") =>
+                destinationConnector.sendMessage(goodsReleasedXml, value.arrivalId, value.version, "IE025")
+                Future.successful(Redirect(routes.ResponseController.onPageLoad()))
+              case "3" =>
+                destinationConnector.sendMessage(unloadingPermissionWithSealsXml, value.arrivalId, value.version, "IE043")
+                Future.successful(Redirect(routes.ResponseController.onPageLoad()))
+              case "4" =>
+                destinationConnector.sendMessage(unloadingPermissionWithoutSealsXml, value.arrivalId, value.version, "IE043")
+                Future.successful(Redirect(routes.ResponseController.onPageLoad()))
+              case _ =>
+                ??? //todo: error out
```

Comment:
Maybe a BadRequest is enough for what this is for?

Reply:
should we maybe do this as an error and have a form constraint to check this can be posted., we can then pull the redirect to the end rather than have this repeated throughout?

Reply:
you could but it's probably overkill for now